composeApp/src/commonMain/kotlin/com/overdrive/cruiser/views/MySpotsView.kt

```diff
@@ -57,134 +75,264 @@ fun MySpotsView(mySpotsViewModel: MySpotsViewModel, onAddSpotClick: () -> Unit)
+                                ) {
+                                    Row(
+                                        modifier = Modifier.padding(16.dp),
+                                        verticalAlignment = Alignment.CenterVertically
+                                    ) {
+                                        Text("5$/hour")
+                                    }
+                                }
+                                Spacer(modifier = Modifier.width(8.dp))
+                                SpotOnField(
+                                    modifier = Modifier
+                                        .height(55.dp)
+                                        .weight(0.45f)
+                                        .shadow(8.dp, RoundedCornerShape(12.dp))
+                                        .background(
+                                            color = Color.White,
+                                            shape = RoundedCornerShape(12.dp)
+                                        )
+                                        .clip(RoundedCornerShape(12.dp))
+                                ) { //
+                                    Row(
+                                        modifier = Modifier.padding(16.dp),
+                                        verticalAlignment = Alignment.CenterVertically
+                                    ) {
+                                        Text("50$/day")
+                                    }
+                                }
+                            }
+                            Spacer(modifier = Modifier.height(8.dp))
 
-                        Row(modifier = Modifier.align(Alignment.Center)) {
-                            RatingStarView(4)
-                        }
 
-                        Icon(
-                            modifier = Modifier.size(24.dp).align(Alignment.CenterEnd),
-                            imageVector = Icons.Filled.Share,
-                            contentDescription = "Share",
-                            tint = Color.Black
-                        )
-                    }
+                            SpotOnField(modifier = Modifier.fillMaxWidth()) { // Popularity
+                                Row(
+                                    modifier = Modifier.padding(16.dp),
+                                    verticalAlignment = Alignment.CenterVertically
+                                ) {
+                                    Text("226 total bookings")
+                                }
+                            }
+                            SpotOnField(modifier = Modifier.fillMaxWidth()) { // User's feedback
+                                Row(
+                                    modifier = Modifier.padding(16.dp),
+                                    verticalAlignment = Alignment.CenterVertically
+                                ) {
+                                    Text("Check feedback from other users >")
```

Comment:
Should probably use the arrow icon from Icons.AutoMirrored.Default.ArrowForward here and make this a button so it's consitent with other buttons. This will follow a similar format to the buttons on the profile view